examples/resources/pingfederate_local_identity_identity_profile/resource.tf

```diff
@@ -1,6 +1,23 @@
-resource "pingfederate_local_identity_identity_profile" "localIdentityIdentityProfilesExample" {
+terraform {
```

Comment:
Need to remove these terraform and provider blocks from any example that have them